workday/16.2/modules/ROOT/pages/workday-connector-studio.adoc

```diff
@@ -133,6 +133,26 @@ This parameter supports expressions, which allow the value of this parameter to
 |User Name |Enter the username with which to log in.
 |===
 
+[[oauth-authentication]]
+=== Workday OAuth Code Authentication Connection
+
+include::connectors::partial$oauth2-description.adoc[]
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure OAuth2 authentication:
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Connection |Select `Workday OAuth Code Authentication Connection`.
+|Tenant Name |Enter the Workday Tenant ID, which usually has the suffix `pt1` appended to it, such as `acme_pt1`.
```

Comment:
we can remove this "which usually has the suffix `pt1` appended to it, such as `acme_pt1`"